featuretools/selection/selection.py

```diff
@@ -18,3 +18,115 @@ def remove_low_information_features(feature_matrix, features=None):
                     if f.get_name() in feature_matrix.columns]
         return feature_matrix, features
     return feature_matrix
+
+
+def remove_highly_null_features(feature_matrix, features=None, pct_null_threshold=0.95):
+    """
+        Removes columns from a feature matrix that have higher than a set threshold
+        of null values.
+
+        Args:
+            feature_matrix (:class:`pd.DataFrame`): DataFrame whose columns are feature names and rows are instances.
+            features (list[:class:`featuretools.FeatureBase`] or list[str], optional): List of features to select.
+            pct_null_threshold (float): If the percentage of NaN values in an input feature exceeds this amount,
+                    that feature will be considered highly-null. Defaults to 0.95.
+
+        Returns:
+            pd.DataFrame, list[:class:`.FeatureBase`]:
+                The feature matrix and the list of generated feature definitions. Matches dfs output.
+                If no feature list is provided as input, the feature list will not be returned.
+    """
+    if pct_null_threshold < 0 or pct_null_threshold > 1:
+        raise ValueError("pct_null_threshold must be a float between 0 and 1, inclusive.")
+
+    percent_null_by_col = (feature_matrix.isnull().mean()).to_dict()
+
+    if pct_null_threshold == 0.0:
+        keep = [f_name for f_name, pct_null in percent_null_by_col.items() if pct_null <= pct_null_threshold]
+    else:
+        keep = [f_name for f_name, pct_null in percent_null_by_col.items() if pct_null < pct_null_threshold]
+
+    return _apply_feature_selection(keep, feature_matrix, features)
+
+
+def remove_single_value_features(feature_matrix, features=None, count_nan_as_value=False):
+    """Removes columns in feature matrix where all the values are the same.
+
+        Args:
+            feature_matrix (:class:`pd.DataFrame`): DataFrame whose columns are feature names and rows are instances.
+            features (list[:class:`featuretools.FeatureBase`] or list[str], optional): List of features to select.
+            count_nan_as_value (bool): If True, missing values will be counted as their own unique value.
```

Comment:
> If True, missing values will be counted as their own unique value.  If set to True, a feature that has one unique value and all other data is missing will be counted as only having a single unique value.

These sentences seem at odds with each other

Reply:
Hmm yeah, definitely contradictory. How about:

> If True, missing values will be counted as their own unique value. If set to True, a feature that has one unique value and all other data missing will be removed from the feature matrix.

Reply:
Yeah, that sounds good